test/edu-benefits/pages/applicantInformation.unit.spec.jsx

```diff
@@ -142,12 +148,16 @@ describe('Edu 1990e applicantInformation', () => {
 
 describe('Edu 5490 applicantInformation', () => {
   // Make sure these match the fields listed in the 5490
-  const { schema, uiSchema } = applicantInformation(fullSchema5490, [
-    'relativeFullName',
-    'relativeSocialSecurityNumber',
-    'relativeDateOfBirth',
-    'gender'
-  ]);
+  const { schema, uiSchema } = applicantInformation(fullSchema5490, {
```

Comment:
I dislike this pattern for these unit tests. We're only coincidentally testing the actual forms and this test in particular had code that wasn't being used (the field list.